target/scripts/helpers/utils.sh

```diff
@@ -5,8 +5,16 @@ function _escape
   echo "${1//./\\.}"
 }
 
-# Check if string input is an empty line, only whitespaces
-# or `#` as the first non-whitespace character.
+# Returns input after filtering out lines that are:
+# empty, white-space, comments (`#` as the first non-whitespace character)
+function _filter_to_valid_lines
```

Comment:
Maybe one of these fits better?

```suggestion
function _get_valid_lines
```

```suggestion
function _get_valid_lines_from_file
```

Reply:
Sure, I'll make that change in the follow-up PR :+1: